examples/CSharp/OpenCL Tutorials/HelloWorld/HelloWorld.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can you move this project to the `OpenCL Demos` folder until we figure out what a formal set of tutorials looks liek for OpenCL?